app/controllers/dashboard_controller.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 require "velum/kubernetes"
+require "velum/suse_connect"
 
 # DashboardController shows the main page.
 class DashboardController < ApplicationController
+  skip_before_action :authenticate_user!, only: :autoyast
```

Comment:
Yes, my guess is that we should add a restriction of what IP ranges can actually request this autoyast configuration, for example, but this is beyond the beta1 scope.